presto-parser/src/main/java/com/facebook/presto/sql/tree/RenameTable.java

```diff
@@ -42,6 +42,12 @@ private RenameTable(Optional<NodeLocation> location, QualifiedName source, Quali
         this.target = requireNonNull(target, "target name is null");
     }
 
+    @Override
+    public QualifiedName getQualifiedName()
```

Comment:
How would you return connector id from DDL statement (parser classes)?
I decided to use `createQualifiedObjectName` from `MetadataUtil` because it's used in other places so we avoid code duplication and minimize chances the catalog resolution will differ for notification and other purposes (e.g. in `StatementAnalyzer`).
If you have better idea how to retrieve catalogName/connectorId based on `Statement` and `Metadata` I'd be more than happy to consider it in place of what I've done here. :)